specification/resources/functions/functions_delete_namespace.yml

```
@@ -0,0 +1,44 @@
+operationId: functions_delete_namespace
+
+summary: Delete Namespace
+
+description: >-
+  Deletes the given namespace.  When a namespace is deleted all assets, in the namespace are deleted, this
+  includes packages, functions and triggers. Deleting a namespace is a destructive operation and assets in the
+  namespace are not recoverable after deletion. Some metadata is retained, such as activations, or soft deleted for reporting purposes.
+
+  To delete namespace, send a DELETE request to `/v2/functions/namespaces/{{namespace_id}}`.
```

The rest of the spec uses the following pattern in docs. It would be good to keep this consistent.

```suggestion
  To delete namespace, send a DELETE request to `/v2/functions/namespaces/$NAMESPACE_ID`.
```